pkg/interactive/inputs/request_loader/loader.go

```diff
@@ -0,0 +1,108 @@
+package requestloader
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/apono-io/apono-cli/pkg/aponoapi"
+	"github.com/apono-io/apono-cli/pkg/clientapi"
+	"github.com/apono-io/apono-cli/pkg/services"
+
+	"github.com/charmbracelet/bubbles/spinner"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/gookit/color"
+)
+
+const (
+	abortingText = "Aborting..."
+)
+
+func (m model) Init() tea.Cmd {
+	return tea.Batch(waitForRequest(m.ctx, m.client, m.creationTime, m.timeout), m.spinner.Tick)
+}
+
+func (e errMsg) Error() string { return e.err.Error() }
+
+func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
```

Comment:
as far as I understand it's now waiting between check intervals, so in case of long wait there could be hundreds of calls to the server. can we add a sleep between retries?

Reply:
added interval of 1 second